packages/@ama-mcp/otter/src/metadata-per-release/index.ts

```diff
@@ -162,10 +166,52 @@ const retrieveAllMetadata = async (
   }));
 };
 
+const matchPackageName = (key: string, packageName: string) => {
+  const packageNameKey = getPackageAndTagFromKey(key)[0]?.toLowerCase();
+  const packageNameLowerCase = packageName.toLowerCase();
+  return packageNameKey.includes(packageNameLowerCase) || packageName.includes(packageNameKey);
+};
+
+const matchTagName = (key: string, tagName: string) => {
+  const tagNameKey = getPackageAndTagFromKey(key)[1]?.toLowerCase().replace(/^v/, '');
```

Comment:
```suggestion
  const tagNameKey = getPackageAndTagFromKey(key)[1]?.toLowerCase().replace(/^v/i, '');
```
(just in case)

Or `semver.valid()` if you have semver helpers

Reply:
At first, I didn't want to add `semver` but maybe at some point it make sense to add it.
I fixed the regexp as you proposed, we may want to open an issue to use `semver` in the different tools link to version already created.

Reply:
I'm not sure to understand how the `/i` helps since there is a `toLowerCase()` just before

Reply:
Indeed :D
I didn't think about it